Makefile.toml

```diff
@@ -266,9 +266,31 @@ cargo install \
 '''
 ]
 
+# We need Cargo version 1.51 or higher in order to build a workspace's
+# dependency during build-package
+[tasks.check-cargo-version]
+script_runner = "bash"
+script = [
+'''
+set -euo pipefail
+cargo_version=$(cargo --version | awk '{print $2}')
+strarr=(${cargo_version//./ })
+cargo_major="${strarr[0]}"
+cargo_minor="${strarr[1]}"
+if [ "${cargo_major}" -gt "1" ] ; then
```

Comment:
nit. It would be nice if we could standardize on `[[...]]`.
nit. I don't think you need quotes for things without spaces (ie. numbers)

Reply:
This is typically done to maintain some level of compatibility with other shells, and then only use `[[ ]]` where you know you need bash specifics.

Reply:
If we are worried about compatibility with other shells, then shouldn't we avoid _all_ instances of bash-isms?

Reply:
No, we do need some of them.  It's a balance - sometimes bash-isms are the best way to do something, but otherwise staying compatible with POSIX sh and BSD userland is a good habit.  We've seen both of those things bite us in Makefile.toml, for example.  Even if your shebang is bash, it's nice to use sh basics when they're essentially identical, like in this case, in case you need the script to run elsewhere someday.  (I'm old, and so have run into this many times.)